scripts/deploy.sh

```diff
@@ -18,6 +18,8 @@
 set -e -u -x
 set -o pipefail
 
+DEPLOY_FILEBEAT=${DEPLOY_FILEBEAT:-yes}
+DEPLOY_MAAS=${DEPLOY_MAAS:-yes}
```

Comment:
@xgerman instead of setting these values in the gating pre script, I decided to preserve the original behavior of the script by setting default values here.